test/autorest/lrogroup/lrosadsheader_test.go

```diff
@@ -181,17 +151,10 @@ func TestLROSADSBeginPost202NonRetry400(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	poller, err = op.ResumePost202NonRetry400(rt)
-	if err != nil {
-		t.Fatal(err)
-	}
-	res, err := resp.PollUntilDone(context.Background(), 1*time.Millisecond)
```

Comment:
Why not include all the PollUntilDone methods?

Reply:
These operations are intended to fail when you first poll the polling endpoint.  So what happens is, since we now poll in the Resume methods, the Resume calls fail (expected) and return an error.